test/e2e/test_kfservice_creating.py

```diff
@@ -0,0 +1,43 @@
+import time
+from kubernetes import client
+
+from kfserving import KFServingClient
+from kfserving import constants
+from kfserving import V1alpha2EndpointSpec
+from kfserving import V1alpha2PredictorSpec
+from kfserving import V1alpha2TensorflowSpec
+from kfserving import V1alpha2InferenceServiceSpec
+from kfserving import V1alpha2InferenceService
+from kubernetes.client import V1ResourceRequirements
+
+api_version = constants.KFSERVING_GROUP + '/' + constants.KFSERVING_VERSION
+KFServing = KFServingClient(load_kube_config=True)
```

Comment:
no, haha, if without `load_kube_config=True`, will go `load_incluster_config()` since that's in the cluster, instead of `load_kube_config()`. I ran into a probelm that http 404 error while using python API, and resovled by add `load_kube_config=True`.